File: Task3-WebClient/EventSuiteWeb/src/app/api/models/venue-request.ts
/* tslint:disable */
/* eslint-disable */
import { VenueType } from '../models/venue-type';
export interface VenueRequest {
  description?: string | null;
  floor?: number;
  mallId?: number | null;
  maxSize?: number;
  roomNumber?: string | null;
  services?: string | null;
  square?: number | null;
  type?: VenueType;
}

File: apzkr-pzpi-21-3-lukianchuk-danylo/Task1-Server/EventSuite.API/EventSuite.Core/DTOs/Requests/Venue/VenueRequest.cs
using EventSuite.Core.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EventSuite.Core.DTOs.Requests.Venue
{
    public class VenueRequest
    {
        public VenueType Type { get; set; }
        public string? Description { get; set; }
        public double? Square { get; set; }
        public int MaxSize { get; set; }
        public string? Services { get; set; }
        public string? RoomNumber { get; set; }
        public int Floor { get; set; }
        public int? MallId { get; set; }
    }
}
